docs/internals/ffi.rst

```diff
@@ -63,6 +63,18 @@ as well as an extended
 Note: One quirk of the ``.C`` interface is the requirement that all inputs and
 return values are passed by pointers. This is the reason for the ``bridgestan_R`` files in the source.
 
+Rust
+____
+
+The Rust interface uses two crates in addition to the built-in
+`FFI types <https://doc.rust-lang.org/core/ffi/index.html>`__.
+Some general guidance on Rust FFI can be found in the
+`Rust Book <https://doc.rust-lang.org/book/ch19-01-unsafe-rust.html#using-extern-functions-to-call-external-code>`__.
+
+These are `bindgen <https://docs.rs/bindgen>`__, which generates the Rust bindings from the C headers,
+and `libloading <https://docs.rs/libloading>`__, which provides easy dynamic library loading functionality.
+
+Care is taken to provide "safe" Rust wrappers so that users of the crate do not need to use the `unsafe` keyword.
```

Comment:
You need an unsafe function to register a printing function, but I think that's probably not worth mentioning here, given that that's a pretty specialized use-case?

Reply:
Yes, I think that's the exception rather than the rule